test/stat_tracker_test.rb

```diff
@@ -437,112 +449,55 @@ def test_it_gets_rival
 2014021171, 2014020149, 2014020384, 2014020041, 2014020462, 2014020951, 2014021134, 2014020022, 2014020607, 2014020134, 2014020324, 2014021219, 2014020759, 2014020092, 2014020976, 2014020402, 2014020379, 2014020949, 2014020453, 2014020203, 2014020698, 2014021186, 2014020859, 2014020488, 2014020325, 2014020511, 2014020454, 2014020990, 2014020849, 2014020790, 2014020136, 2014020709, 2014020007, 2014020564, 2014020719, 2014021048, 2014021086, 2014021022, 2014021178, 2014020397, 2014020003, 2014020931, 2014020452, 2014020023, 2014020750, 2014020445, 2014020597, 2014020360, 2014020113, 2014021025, 2014021066, 2014020423, 2014020018, 2014020443, 2014020673, 2014020748, 2014020916, 2014020581, 2014020418, 2014020907, 2014020796, 2014020899, 2014020912, 2014020784, 2014021177, 2014021141, 2014020923, 2014020782, 2014020549, 2014020808, 2014020283, 2014020048, 2014020733, 2014020209, 2014021049, 2014020328, 2014020562, 2014020517, 2014020072, 2014020011, 2014020897, 2014020046, 2014020154, 2014020239, 2014020547, 2014020586, 2014020090, 2014020989, 2014020778, 2014021035, 2014020786, 2014020566, 2014020298, 2014021123, 2014020479, 2014020002, 2014020501, 2014020240, 2014020116, 2014020489, 2014020093, 2014020518, 2014020883, 2014020188, 2014020290, 2014021213, 2014020440, 2014020663, 2014020196, 2014020127, 2014020276, 2014020826, 2014020840, 2014020382, 2014020991, 2014021111, 2014021196, 2014020944, 2014020837, 2014021081, 2014020516, 2014021051, 2014020280, 2014020858, 2014020609, 2014020945, 2014021026, 2014020372, 2014021038, 2014021082, 2014020666, 2014020422, 2014020835, 2014020824, 2014020953, 2014021103, 2014021143, 2014020320, 2014021054, 2014021110, 2014020220, 2014020057, 2014021124, 2014021158, 2014021034, 2014020775, 2014021211, 2014020157, 2014020657, 2014020872, 2014020420, 2014020539, 2014020936, 2014020522, 2014021065, 2014021175, 2014020641, 2014020563, 2014020478, 2014020463, 2014020229, 2014021050, 2014020125, 2014020083, 2014021073, 2014021202, 2014021099, 2014020341, 2014020302, 2014020513, 2014020538, 2014020662, 2014020687, 2014020147, 2014020417, 2014021201, 2014021087, 2014021222, 2014021157, 2014020467, 2014020570, 2014020296, 2014020435, 2014020943, 2014021041, 2014020724, 2014020968, 2014020419, 2014020389, 2014020725, 2014020627, 2014021132, 2014020244, 2014020802, 2014021079, 2014020450, 2014021180, 2014020088, 2014020109, 2014020045, 2014020049, 2014020009, 2014021076, 2014020349, 2014020409, 2014020895, 2014020156, 2014020281, 2014021116, 2014020515, 2014020531, 2014021168, 2014020557, 2014021190, 2014020014, 2014020821, 2014020833, 2014020457, 2014021130, 2014020291, 2014020346, 2014020747, 2014020224, 2014020393, 2014020077, 2014020366, 2014020433, 2014021146, 2014020101, 2014020974, 2014020227, 2014021036, 2014020192, 2014020228, 2014020593, 2014020267, 2014020368, 2014020193, 2014020866, 2014021184, 2014021021, 2014020053, 2014020601, 2014021167, 2014020874, 2014020186, 2014020333, 2014020720, 2014020746, 2014020979]
```

Comment:
I think we should write these test differently... Like test if there are a certain # of seasons that come up and test that vs. a .count of the array and test if the there is only one? of each of these season.